package.json

```diff
@@ -337,8 +337,6 @@
     "@typescript-eslint/experimental-utils": "4.22.0",
     "@typescript-eslint/parser": "4.22.0",
     "@types/jest-axe/axe-core": "4.2.1",
-    "@types/react@<18.0.0": "@types/react@17.0.44",
```

Comment:
these resolution declarations are invalid - so they were not doing anything, but because yarn already provides lot of warnings during install, its hard to spot that we provided invalid config. we should create tooling to catch those issues to provide best possible DX